Packs/RegexReplace/Scripts/RegexReplace/RegexReplace.yml

```diff
@@ -7,15 +7,29 @@ args:
   required: true
 - description: Template string to format patterns matched with regex
   name: output_format
-  required: true
 - auto: PREDEFINED
-  defaultValue: "false"
-  description: True if the pattern matching is case insensitive, otherwise
-    False.
+  defaultValue: 'false'
+  description: Whether character matching will be case-insensitive. Default is "false".
   name: ignore_case
   predefined:
-  - "true"
-  - "false"
+  - 'true'
+  - 'false'
+- name: multi_line
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+  description: Process value in multiline mode.  See more information on re.MULTILINE, see https://docs.python.org/3/library/re.html.
+  defaultValue: 'false'
+- name: period_matches_newline
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+  description: Whether to make the '.' character also match a new line. Default is "false".
+  defaultValue: 'false'
+- name: action_dt
+  description: The last action for each matched value to transform
```

Comment:
Not sure what this argument is supposed to do. Update the description and add an example.

Reply:
`action_dt` is intended to do small manipulation to each value.

A prospect wants to transform variable sumbols from %Name% to {name}.
%Name% is bracketed off by %, and Name includes capital letters.
{name} is bracketed off by {}, and name only consists of lower letters.

They expect:
```
Input: Hello %UserName%, Your UserID is %UserID%.
Output: Hello {UserName}, Your UserID is {UserID}.
```
In that case, they can do it with the parameters below using `action_dt` for making lower letters.
```
regex: %(.+?)%
output_format: {\1}
action_dt: .=val.toLowerCase()
```
Another prospect wants to convert logs starting with unix time to logs starting with it in ISO time format.
```
1660872463: TCP Port 80|0|cn1=1 cn1Label=Host ID dvc=hostname
2022-08-19T01:27:43.000Z: TCP Port 80|0|cn1=1 cn1Label=Host ID dvc=hostname
```
In that case, they can do it with the parameters below.
```
regex: ^[0-9]+
output_format: \0
action_dt:.=new Date(val * 1000).toISOString()
```